core/blockly.ts

```diff
@@ -572,17 +572,22 @@ export const PROCEDURE_CATEGORY_NAME: string =
     (Procedures as AnyDuringMigration).CATEGORY_NAME;
 
 // I hate this so much.
-Workspace.prototype.newBlock = function (prototypeName: string, opt_id?: string): Block {
-  return new Block(this, prototypeName, opt_id);
-}
-
-WorkspaceSvg.prototype.newBlock = function (prototypeName: string, opt_id?: string): BlockSvg {
-  return new BlockSvg(this, prototypeName, opt_id);
-}
-
-WorkspaceSvg.newTrashcan = function(workspace: WorkspaceSvg): Trashcan {
-  return new Trashcan(workspace);
-}
+Workspace.prototype.newBlock = function(prototypeName: string, opt_id?: string):
+                                   Block {
+                                     return new Block(
+                                         this, prototypeName, opt_id);
+                                   }
+
+                               WorkspaceSvg.prototype.newBlock =
+    function(prototypeName: string, opt_id?: string):
+        BlockSvg {
+          return new BlockSvg(this, prototypeName, opt_id);
+        }
```

Comment:
Go home clang-format, you're drunk.  (Line 581: WTF?)

Could the tool be persuaded to be slightly less toolish by manually inserting a newline after the `=` in both declarations?  Or… well, please: just see what you can do to mitigate this fiasco.

Reply:
Turned off clang for these lines.